systemtests/src/test/java/io/enmasse/systemtest/plans/PlansTest.java

```diff
@@ -153,6 +157,70 @@ public void testQuotaLimits() throws Exception {
                         Destination.anycast("a4", anycastPlan.getName())
                 ), username, password);
 
+        //check broker limits
+        checkLimits(addressSpace,
+                Arrays.asList(
+                        Destination.queue("q1", queuePlan.getName()),
+                        Destination.queue("q2", queuePlan.getName())
+                ),
+                Collections.singletonList(
+                        Destination.queue("q3", queuePlan.getName())
+                ), username, password);
+
+        checkLimits(addressSpace,
+                Arrays.asList(
+                        Destination.queue("q1", queuePlan.getName()),
+                        Destination.queue("q2", queuePlan.getName()),
+                        Destination.queue("q3", queuePlan2.getName()),
+                        Destination.queue("q4", queuePlan2.getName()),
+                        Destination.queue("q5", queuePlan2.getName()),
+                        Destination.queue("q6", queuePlan2.getName())
+                ), null, username, password);
```

Comment:
I would prefer emptyList instead of null for notAllowedDestination. Will it work with .stream()... correctly?